src/main/java/org/mifos/connector/mpesa/utility/SafaricomUtils.java

```diff
@@ -66,17 +66,17 @@ public BuyGoodsPaymentRequestDTO channelRequestConvertor(TransactionChannelColle
         buyGoodsPaymentRequestDTO.setPartyA(payer);
         buyGoodsPaymentRequestDTO.setPhoneNumber(payer);
 
-        buyGoodsPaymentRequestDTO.setPartyB(tillNumber);
-        buyGoodsPaymentRequestDTO.setBusinessShortCode(businessShortCode);
+        buyGoodsPaymentRequestDTO.setPartyB(mpesaUtils.getTillNumber());
```

Comment:
Utils are not to be confused with DTOs. Utils are not even to be confused with business logic. Utilities assist in writing business logic.